keyboards/crkbd/keymaps/default/keymap.c

```diff
@@ -17,6 +17,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include QMK_KEYBOARD_H
+#include <stdio.h>
```

Comment:
Shouldn't be necessary to include this manually, no?

Reply:
It's needed because of the snprintf function, used for the oled code.